### PR TITLE
Implement new elevation, redesign profile page

### LINKED ISSUE #174: edit profile sections background
fix dark mode colors - there's no contrast right now image

### PR DESCRIPTION
**Pull request by alexzhang1618**
# Info

Closes #174.

## Changes

- Add new CSS vars for elevation
- Recolor/redesign navbar and profile page

# Type of Change

- [ ] Bug Fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
      expected)
- [ ] Logistics Change (A change to a README, description, or dev workflow setup like
      linting/formatting)
- [ ] Continuous Integration Change (Related to deployment steps or continuous integration
      workflows)
- [ ] Other: (Fill In) 

# Testing

I have tested that my changes fully resolve the linked issue ...

- [x] locally on Desktop.
- [ ] on the live deployment preview on Desktop.
- [ ] on the live deployment preview on Mobile.
- [ ] I have added new Cypress tests that are passing.

# Checklist

- [x] I have performed a self-review of my own code.
- [x] I have followed the style guidelines of this project.
- [x] I have documented any new functions in `/src/lib/*` and commented hard to understand areas
      anywhere else.
- [x] My changes produce no new warnings.

# Screenshots

Screenshot 2024-03-24 at 6 32 36 PM
Screenshot 2024-03-24 at 6 33 29 PM

### CODE REVIEW

File: src/components/profile/UserProfilePage/index.tsx [hunk 1]
```diff
@@ -50,33 +48,26 @@ export const UserProfilePage = ({
...
-            </Tooltip>
-          </div>
-          <div className={styles.cardRank}>
-            <Typography variant="h4/regular" className={styles.rank}>
-              {getUserRank(handleUser.points)}
-            </Typography>
+          <div className={styles.userInfo}>
+            <div className={styles.cardName}>
+              <Typography variant="h1/bold" component="h1">
+                {userName}
```
Comment (SheepTester) [inline src/components/profile/UserProfilePage/index.tsx]: maybe name the variable `fullName` beccause when I saw `userName`, I thought that was their handle

File: src/components/profile/UserProfilePage/index.tsx [hunk 2]
```diff
@@ -50,33 +48,26 @@ export const UserProfilePage = ({
...
+              </Typography>
+              <Typography
+                variant="h5/regular"
+                onClick={() => {
+                  copy(window.location.href);
+                  showToast(`Copied link to ${userName}'s profile!`);
```
Comment (SheepTester) [inline src/components/profile/UserProfilePage/index.tsx]: Should it use their first name here? like how line 87 uses

```tsx
          {isSignedInUser ? 'My' : `${handleUser.firstName}'s`} Progress
```

File: src/components/profile/UserProfilePage/index.tsx [hunk 3]
```diff
@@ -106,16 +103,18 @@ export const UserProfilePage = ({
           {isSignedInUser ? 'You need ' : `${handleUser.firstName} needs `}
           {100 - (handleUser.points % 100)} more points to level up to
```
Comment (SheepTester) [inline src/components/profile/UserProfilePage/index.tsx]: Should we hide this when you're at the last rank?

image

**Review by SheepTester — APPROVED** [review]
profile page looks good, should other cards get the same treatment?

image

image

image

image

**Comment by SheepTester** [comment]
also why does this close #174 when it still looks like this

image

**Comment by alexzhang1618** [comment]
> profile page looks good, should other cards get the same treatment?

Yep, I'll do it in a future PR

File: src/components/profile/UserProfilePage/index.tsx (hunk 3 above)
Comment (alexzhang1618) [reply src/components/profile/UserProfilePage/index.tsx]: I'll make it say Level 1009 instead 👍

**Comment by alexzhang1618** [comment]
> also why does this close https://github.com/acmucsd/membership-portal-ui-v2/issues/174 when it still looks like this

fixed

**Pull request merged**